examples/mvdream/models/mvdream.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any, Dict, List, Union
+
+import numpy as np
+import torch
+
+from mvdream.camera_utils import get_camera
+from mvdream.ldm.models.diffusion.ddim import DDIMSampler
+from mvdream.model_zoo import build_model
+
+class MVDreamModel:
+    def __init__(self, model_name: str = "sd-v2.1-base-4view", dtype: str = "float16"):
+
+        if torch.cuda.is_available():
+            self.device_str = "cuda"
+        else:
+            self.device_str = "cpu"
+        self.torch_dtype = getattr(torch, dtype)
+
+        self.model = build_model(model_name)
+        self.model.device = self.device_str
+        self.model.to(self.device_str)
+        self.model.eval()
+
+        self.sampler = DDIMSampler(self.model)
+
+        # output 4 view with fixed angles
+        self.nviews = 4
+        self.camera = get_camera(self.nviews).to(self.device_str)
+
+    def __call__(
+        self,
+        prompts: Union[str, List[str]],
+        negative_prompts: Union[str, List[str]] = [""],       
```

Comment:
[nit] Defaults to `None` which is identical to the the diffusers library. See how we do this in [stable_diffusion.py](https://github.com/autonomi-ai/nos/blob/main/nos/models/stable_diffusion.py).

Reply:
True, changed to `None` now.